docs/reference/api/usb/USBMouseKeyboard.md

```diff
@@ -0,0 +1,41 @@
+## USBMouseKeyboard
+
+<span class="images">![](https://os.mbed.com/docs/v5.9/feature-hal-spec-usb-device-doxy/class_u_s_b_mouse_keyboard.png)<span>USBMouseKeyboard class hierarchy</span></span>
+
+You can use the USBMouseKeyboard interface to emulate a mouse and a keyboard at the same time over the USB port. You can send both presses and mouse movements with this class.
```

Comment:
What are some best practices for using USBKeyboard? What are some concepts you need to understand to use it?